apm-agent-plugins/apm-jdbc-plugin/src/test/java/co/elastic/apm/agent/jdbc/JdbcDbIT.java

```diff
@@ -43,7 +43,8 @@ public static Iterable<Object[]> data() {
             {"jdbc:tc:postgresql:9://hostname/databasename", "postgresql", "databasename", true},
             {"jdbc:tc:postgresql:10://hostname/databasename", "postgresql", "databasename", true},
             {"jdbc:tc:mariadb:10://hostname/databasename", "mariadb", "databasename", true},
-            {"jdbc:tc:sqlserver:2017-CU12://hostname/databasename", "mssql", "master", false}, // for mssql the 'master' name comes from the runtime catalog fallback
+            // SQL Server image seems to be broken with recent kernel versions: https://github.com/microsoft/mssql-docker/issues/868
+            //{"jdbc:tc:sqlserver:2017-CU12://hostname/databasename", "mssql", "master", false}, // for mssql the 'master' name comes from the runtime catalog fallback
```

Comment:
I don't think there is a significant risk to disable this test case, maybe add a TODO prefix to make it clear that it might be something we are looking to fix later.

Reply:
Added in [b9e4b85](https://github.com/elastic/apm-agent-java/pull/3952/commits/b9e4b85d89b5846a8157027201dfe79617418cff)